gui/functionality/add_cluster.py

```diff
@@ -54,7 +55,9 @@ def handle_complete_add_cluster(self):
         self.popup_window.show_message("The cluster has been successfully added", WindowType.success)
 
     def handle_error_add_cluster(self, exception):
-        if isinstance(exception, ValueError):
+        if isinstance(exception, EmptyFieldError):
```

Comment:
How can a cluster name be empty?

Reply:
Becasue somebody can just try to break our application :P

Reply:
Is it possible to add a cluster with an empty name?

Reply:
No but the error for this was just an empty string. So I decided to make it more verbose.

Reply:
So when can this error happen if there is no cluster with an empty name?

Reply:
It's in adding cluster. It doesn't have the clusters list. Just remove the name.

Reply:
ok

Reply:
Plus if it would be a cluster list, if you don't have any clusters the story is the same